Move storage controllers into `storage` folder in `control_strategies` (#678)
NREL/GreenHEART: docs/storage/storage_models_index.md

Reviewer: ```suggestion
Storage technologies input and output the 'Storage Commodity' (`commodity`) as a time series. These technologies can be both filled or charged and unfilled or discharged, resulting in a commodity stream that can be positive and negative. These models are usually constrained by two key model parameters: storage capacity and charge/discharge rate.
```

Reviewer: ```suggestion
H2I currently supports the following storage performance models:
- [`StoragePerformanceModel`](#simple-generic-storage-performance)
```

- `annual_commodity_produced`: each value is the `total_commodity_produced` scaled to a 1-year (8760 hours) simulation. This value may be negative if the storage charges more than discharges.

> - `capacity_factor`: the storage capacity factor calculated as capacity factors are calculated in converter technologies, a ratio of the sum of *`total_commodity_produced`* to the discharge rate (or `rated_commodity_production`). This value may be negative if the storage charges more than discharges.

Reviewer: ```suggestion
- `capacity_factor`: the storage capacity factor which is calculated in the same way that capacity factors are calculated in converter technologies, which is a ratio of the sum of *`total_commodity_produced`* to the discharge rate (or `rated_commodity_production`). This value may be negative if the storage charges more than discharges.
```
